src/components/react_canvas/reactive_chart.tsx

```diff
@@ -349,8 +349,16 @@ class Chart extends React.Component<ReactiveChartProps, ReactiveChartState> {
       chartRotation,
       chartTransform,
       setCursorPosition,
+      isChartEmpty,
     } = this.props.chartStore!;
 
+    if (isChartEmpty) {
+      return (
+        <div className="echUnavailable">
+          <h3>No series selected to display</h3>
```

Comment:
Try not to use actual header level tags here. For accessibility, it's vital that the hierarchy of h1-h6 is correct within the DOM order and being that this is a plugin library, you can't really be sure where this component will fall within the DOM order. Best to just leave this is a `<p>` tag.

Reply:
I would just make this sentence more generic like:
```suggestion
          <p>No data to display</p>
```

Reply:
Hi @cchaos thanks for the feedback! Let me know what you think about the changes in commit 6719484 

For instance, I wasn't positive about adding the chart icon over or just keeping it to the text message. I'm open to what you think might make the most sense. Thanks!

Reply:
Agreed, it doesn't need any icon (at least not at the moment).